epoxy-processor/src/main/java/com/airbnb/epoxy/GeneratedModelWriter.kt

```diff
@@ -435,6 +444,18 @@ internal class GeneratedModelWriter(
         }
     }
 
+    private fun generateVisibilityMethods(modelInfo: GeneratedModelInfo): Iterable<MethodSpec> {
+        val methods = ArrayList<MethodSpec>()
+        val boundObjectParam = ParameterSpec.builder(modelInfo.modelType, "object", FINAL).build()
```

Comment:
any overhead on adding all of this ? Can it be optional ?

Reply:
This code get executed at build time, no overhead at runtime. Is it answering your comment ?